### PR TITLE
Update clinvar import + post process scripts

### PR DESCRIPTION
**Pull request by dglemos**
By default, the plugin prints many messages, making it very difficult to find serious issues/errors.
This update adds most of the messages to be printed in debug mode. It also:
- updates from `|| die` to `or die`
- fix indentation
- updates method `clean_old_data`

### CODE REVIEW

File: scripts/import/clinvar_post_process.pl [hunk 1]
```diff
@@ -126,7 +126,7 @@ sub run_variation_checks {
         #the expectation is that any new rsID that has to be inserted is a new one (not in dbSNP import)
         #given that approx. 650mil variation is already known, the new rsID should have a bigger number than that
         my $number = $var{name}  =~ s/rs//r;
-        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if $number < 650000000;
+        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if $number < 650000000 if $DEBUG == 1;
```
Comment (likhitha-surapaneni) [inline scripts/import/clinvar_post_process.pl:129]:
```suggestion
        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if($number < 650000000 and  $DEBUG == 1);
```

File: scripts/import/clinvar_post_process.pl [hunk 2]
```diff
@@ -404,55 +407,56 @@ sub check_counts{
...
+  my %class_count;
+  print "\nGetting counts by class\n";
+  $class_count_ext_sth->execute()||die;
+  my $class_num = $class_count_ext_sth->fetchall_arrayref();
+  foreach my $l(@{$class_num}){
+    print "$l->[1]\t$l->[0]\n";
+    $class_count{$l->[0]} = $l->[1];
+  }
+
+  $class_ext_sth->execute()||die;
```
Comment (likhitha-surapaneni) [inline scripts/import/clinvar_post_process.pl:441]:
```suggestion
  $class_ext_sth->execute() or die;
```

File: scripts/import/clinvar_post_process.pl [hunk 3]
```diff
@@ -404,55 +407,56 @@ sub check_counts{
...
+    print "$set_count->[0]->[0] variation entries in $set set\n";
+  }
+
+  $featless_count_ext_sth->execute() or die;
+  my $featless =  $featless_count_ext_sth->fetchall_arrayref();
+  print "$featless->[0]->[0] phenotype entries have no phenotype features\n";
+
+  my %class_count;
+  print "\nGetting counts by class\n";
+  $class_count_ext_sth->execute()||die;
```
Comment (likhitha-surapaneni) [inline scripts/import/clinvar_post_process.pl:434]:
```suggestion
  $class_count_ext_sth->execute() or die;
```

File: scripts/import/clinvar_post_process.pl [hunk 4]
```diff
@@ -404,55 +407,56 @@ sub check_counts{
...
+    print "\nNo variants with class: $l->[0]\n"  unless defined $class_count{$l->[0]} ;
+  }
+
+  #get OMIM variant set count:
+  $omim_set_ext_sth->execute( "ph_omim"); #get OMIM set id
+  my $omim_set_count = $omim_set_ext_sth->fetchrow_array();
+  print "OMIM set variants: ", $omim_set_count, "\n";
+
+  print "Getting ClinVar phenotype_attrib counts\n";
+  $pheno_attrib_ext_sth->execute()||die;
```
Comment (likhitha-surapaneni) [inline scripts/import/clinvar_post_process.pl:453]:
```suggestion
  $pheno_attrib_ext_sth->execute() or die;
```

**Review by likhitha-surapaneni — CHANGES_REQUESTED** [review]

File: scripts/import/clinvar_post_process.pl (hunk 1 above)
Comment (likhitha-surapaneni) [reply scripts/import/clinvar_post_process.pl:129]:
```suggestion
        warn "WARNING: clinvar rsID less than 650mil, likely typo! $var{name} \n" if($number < 650000000 &&  $DEBUG == 1);
```
if we have a preference for && over and

Comment (dglemos) [reply scripts/import/clinvar_post_process.pl:129]: Thanks for pointing that out. This is mistake.

**Review by likhitha-surapaneni — APPROVED** [review]

**Pull request merged**